exasol_script_languages_developer_sandbox/cli/options/vm_options.py

```diff
@@ -0,0 +1,13 @@
+import click
+
+from exasol_script_languages_developer_sandbox.lib.vm_disk_image_format import default_vm_disk_image_formats, \
+    all_vm_disk_image_formats
+
+vm_options = [
+              click.option('--vm-image-format', default=default_vm_disk_image_formats(),
+                             type=click.Choice(all_vm_disk_image_formats()), multiple=True,
+                             help="The VM image format. Can be declared multiple times."),
+              click.option('--no-vm', is_flag=True, help="If set, no vm image will be exported. "
+                                                         "Overrides 'vm-image-format."),
```

Comment:
what do you mean with overrides

Reply:
It means that if this option is set any option of type `vm-image-format` will be ignored (because no vm image at all will be generated).

Reply:
I have changed the help-comment slightly.